inst/schema/spec.md

```diff
@@ -515,13 +515,28 @@ Request
 ```
 
 Response
+
 ```json
 {
   "status": "success",
   "errors": null,
   "data": {
     "workflow_key": "aghast_wolf",
-    "reports": ["deific_thrip", "monarchistic_blackmamba"]
+    "reports": [
+      {
+        "name": "other",
+        "params": {
+          "nmin": [0.5]
+        },
+        "key": "deific_thrip",
+        "execution_order": 1
+      },
+      {
+        "name": "minimal",
```

Comment:
ah, is this the idea behind the list() construction I commented on elsewhere? To remove null elements? We've also done that with a `drop_null` function (something like `function(x) x[!vlapply(x, is.null)])`, which can make intent clearer)

Reply:
yeah, slightly refactored to share logic with existing serialization method: https://github.com/vimc/orderly.server/pull/96/commits/3653a0acab075bb12d181c21dd64be1f018a1352